test/e2e/testdrivers/1.22.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+StorageClass:
+  FromName: true
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: dobs.csi.digitalocean.com-dev
+  SupportedSizeRange:
+    Max: 100Gi
+    Min: 1Gi
+  Capabilities:
```

Comment:
@timoreimann How do I know which Capabilities need to be provided?

Reply:
That's a bit non-trivial, unfortunately: you basically need to know which capabilities are available in CSI (by reading up on the [CSI docs](https://kubernetes-csi.github.io/docs/) and the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md)), which of those our driver supports, and then take a good guess or look at the upstream e2e code to understand what the individual capabilities refer to exactly.

In this case, it's relatively easy since the caps don't seem to have changed between [1.21](https://github.com/kubernetes/kubernetes/blob/release-1.21/test/e2e/storage/framework/testdriver.go) and [1.22](https://github.com/kubernetes/kubernetes/blob/release-1.22/test/e2e/storage/framework/testdriver.go), so we should be fine copying what we have for 1.21 already.

Reply:
Ahh.. I was afraid of that. Guess I got lucky though 😅